ocs_ci/ocs/cluster.py

```diff
@@ -2611,6 +2617,28 @@ def compare_percent_data_from_pvc(self, pvc_obj, data_size):
                 f"✅ Pvc {pvc_obj.name} utilization is {pvc_data_percent} and expected is {pvc_expected_data_percent}"
             )
 
+    def get_lvm_pods_names(self):
```

Comment:
get_lvm_pod_names

Reply:
I don't really understand your point. We have multiple pods check so why use pod? So I'm getting pods names. Logically the name seems right.